docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -2178,6 +2178,60 @@ The third example will autoregister the device and assign the role staff to each
 
 You can have a look in the file vlan_filters.conf, there are some examples on how to use and define filters.
 
+Radius Filter Definition
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+We added the ability to specify filters directly in the portion of code that return the radius answer or do a call to the API.
+
+These rules are only available in one scope:
+
+  returnRadiusAccessAccept
+
+And can be defined using different criteria like:
+
+  node_info.attribute (like node_info.status)
+  switch
+  ifIndex
+  mac
+  connection_type
+  username
+  ssid
+  time
+  owner.attribute (like owner.pid)
+  radius_request.attribute (like radius_request.Calling-Station-Id)
+  violation
+  user_role
+  vlan
+
+
+For example, lets define a rule that return Access Accept when the connection is Ethernet-EAP and when there is no violation
+(merge_return mean that the original answer of PacketFence doesn't have to be merged with the filter answer):
```

Comment:
I'd say it the other way : 
merge_return means that the original answer of PacketFence will be merge with the filter answer automatically